src/crimson/osd/pg_backend.cc

```diff
@@ -677,6 +677,26 @@ seastar::future<> PGBackend::omap_set_header(
   return seastar::now();
 }
 
+seastar::future<> PGBackend::omap_remove_range(
+  ObjectState& os,
+  const OSDOp& osd_op,
+  ceph::os::Transaction& txn)
+{
+  std::string key_begin, key_end;
+	try {
```

Comment:
is this indent intentional? it looks strange.

Reply:
It doesn't look like that in my code locally, that is strange

Reply:
Tab instead of spaces. Perhaps your editor uses 2 spaces-per-stab but this still is strange. Let's just switch to 2 spaces.